src/Option/OptionFactory.ts

```diff
@@ -4,6 +4,8 @@ import { optionFromMatch } from "./Option";
 import { Option } from "./OptionTypes";
 import { OptionPromise } from "./OptionPromiseTypes";
 
+import {some, none} from "./HKTOption/Option"
```

Comment:
I'll remove this later today. I was checking test results and performance differences.